src/main/scala/nl/knaw/dans/easy/stage/fileitem/EasyStageFileItem.scala

```diff
@@ -121,6 +121,7 @@ object EasyStageFileItem {
       _            <- writeFoxml(sdoDir, foxmlContent)
       fmd          <- EasyFileMetadata(s)
       _            <- writeFileMetadata(sdoDir, fmd)
+      _            <- s.isMendeley.filter(b => !b).flatMap(_ => s.file.map(copyFile(sdoDir, _))).getOrElse(Success(Unit))
```

Comment:
I missed the import, then it warned copyFile returns a unit. Actually it throws IO exceptions, you should wrap it in a try.